ci/docker/daml-sdk/Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM openjdk:8u212-alpine
-RUN apk add --no-cache curl bash
+FROM adoptopenjdk:8-jdk-hotspot
 ARG VERSION
 # This is needed to get the DNS requests
 # from Haskell binaries to succeed.
 # Otherwise they fail to even resolve localhost.
 RUN echo 'hosts: files dns' > /etc/nsswitch.conf
-RUN addgroup -S daml && adduser -S daml -G daml
+RUN addgroup --system daml && adduser --system --ingroup daml daml
```

Comment:
Why do we need to switch to system groups?

Reply:
As far as I can tell from the docs that's what the `-S` did on Alpine; this was an attempt at a direct translation.

Reply:
Oh I missed the fact that we had `-S` before. Makes sense :+1: